bundler/spec/other/ext_spec.rb

```diff
@@ -34,19 +34,21 @@
     expect(generic(pl("x86-mswin32"))).to eq(pl("x86-mswin32"))
   end
 
-  it "converts 32-bit mingw platform variants into x86-mingw32" do
-    expect(generic(pl("mingw32"))).to eq(pl("x86-mingw32"))
-    expect(generic(pl("i386-mingw32"))).to eq(pl("x86-mingw32"))
-    expect(generic(pl("x86-mingw32"))).to eq(pl("x86-mingw32"))
+  it "converts 32-bit mingw platform variants into universal-mingw" do
+    expect(generic(pl("i386-mingw32"))).to eq(pl("universal-mingw"))
+    expect(generic(pl("x86-mingw32"))).to eq(pl("universal-mingw"))
```

Comment:
Pending discussion above, I think 32-bit should not be `universal`.
I also think we need a case that `-msvcrt` does not select `universal-mingw`, but does select `universal-mingw-msvcrt`